.travis.yml

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
Is this from an incomplete merge?

Reply:
Yes, these are all from incomplete merges. I didn't know which one to go with. I'll just go with the cookiecutter version and if you want I can always change it.